core/themes/basis/css/skin.css

```diff
@@ -1014,6 +1014,16 @@ ul.secondary li a:active {
 }
 
 /**
+ * Unpublished indicator.
+ */
+.unpublished-indicator {
+  line-height: 1.6rem;
+  background-color: #ee3d23;
+  color: #fff;
+  padding: 0 5px;
```

Comment:
I think @wesruv would prefer you not introduce nasty `px` into his wonderful `em` based theme. :wink: